client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDriver.java

```diff
@@ -91,6 +91,9 @@ public Connection connect(String url, Properties info)
     public boolean acceptsURL(String url)
             throws SQLException
     {
+        if (url == null) {
+            throw new SQLException("URL is null");
```

Comment:
Can you add a test in `TestJdbcVendorCompatibility`?

Reply:
PostgreSQL throws NPE, contrary to the specification. I'm not sure what there is to be gained from this test, since the Javadoc for `java.sql.Driver` says

> Throws: SQLException – if a database access error occurs or the url is null

Our previous behavior is to throw NPE, which violates the specification, and we now test this for our driver in `TestTrinoDriver`.